### ISSUE TITLE
Make transaction lines more compact

### ISSUE THREAD

**Issue by novoselt** [sagemathinc/cocalc#7051]
This is a screenshot of CoCalc window maximized on a 27" monitor:
![image](https://github.com/sagemathinc/cocalc/assets/1993227/40c1bfc6-1d9c-45ca-bc9e-49c21055f59f)
I can only see FOUR transactions! Given that compute servers can generated quite a few of them, I would like to see way more, especially on a big screen. I suspect 20 can easily be displayed with all details and perhaps more.

Possible adjustments:
1) reduce vertical padding which seems excessive, alternating white/light grey lines would be a good option too
2) reduce the number of lines used by some elements.
3) Period start and end probably should be on different lines, but maybe duration can be displayed in the "Time" column on the second line, instead of "Period" on the third?
4) "Active" is a bit redundant since active periods have "now" as their end. Maybe get rid of "now" and instead put "Active" there? It is nice that it stands out more than just a text. Or keep "now" as is and move "Active" to the right from it, on the same line.
5) Is it possible to make the transaction listing vertically resizable or have a button to expand it to the whole screen?

**Comment by williamstein** [comment]
> or have a button to expand it to the whole screen?

I most like this suggestion, or something similar.

**Comment by williamstein** [comment]
I am much less enthusiastic about messing with padding in the table, etc.  This is all imposed by antd, and the entire point of antd is to provide a consistent style across the UI that has been chosen by professional designers.  See https://ant.design/docs/spec/introduce/

If you start tweaking/squishing it, that sort of defeats the purpose.

**Comment by novoselt** [comment]
Agreed, however apart from padding active "Period" entries can be displayed using only 2 lines rather than 4, which makes at least the text space half the height!
